rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -1282,3 +1304,195 @@ fn open_maps_folder() {
+    let time_of_day = world_references.get_field("TimeOfDay").unwrap::<ObjectWrapper>();
+    let light = time_of_day.get_field("Light").unwrap::<ObjectWrapper>();
+    let fun = light.class().find_function("SetBrightness").unwrap();
+    let params = fun.create_argument_struct();
+    params.get_field("NewBrightness").unwrap::<&Cell<f32>>().set(value);
+    unsafe {
+        fun.call(light.as_ptr(), &params);
+    }
+}
+#[rebo::function("Tas::set_sky_light_intensity")]
+fn set_sky_light_intensity(value: f32) {
+    let obj = unsafe { ObjectWrapper::new(UMyGameInstance::get_umygameinstance() as *mut UObject) };
+    let world_references = obj.get_field("WorldReferences").unwrap::<ObjectWrapper>();
+    let time_of_day = world_references.get_field("TimeOfDay").unwrap::<ObjectWrapper>();
+    let light = time_of_day.get_field("Light").unwrap::<ObjectWrapper>();
+    let light_component = light.get_field("LightComponent").unwrap::<ObjectWrapper>();
+    let fun = light_component.class().find_function("SetIntensity").unwrap();
+    let params = fun.create_argument_struct();
+    params.get_field("NewIntensity").unwrap::<&Cell<f32>>().set(value);
+    unsafe {
+        fun.call(light_component.as_ptr(), &params);
+    }
+}
+#[rebo::function("Tas::set_stars_brightness")]
+fn set_stars_brightness(value: f32) {
+    let obj = unsafe { ObjectWrapper::new(UMyGameInstance::get_umygameinstance() as *mut UObject) };
+    let world_references = obj.get_field("WorldReferences").unwrap::<ObjectWrapper>();
+    let time_of_day = world_references.get_field("TimeOfDay").unwrap::<ObjectWrapper>();
+    let stars_brightness = time_of_day.get_field("StarsBrightness").unwrap::<ObjectWrapper>();
+    let float_curves = stars_brightness.get_field("FloatCurve").unwrap::<StructValueWrapper>();
+    let keys = float_curves.get_field("Keys").unwrap::<ArrayWrapper<StructValueWrapper>>();
+    keys.get(0).unwrap().get_field("Value").unwrap::<&Cell<f32>>().set(value);
+}
+#[rebo::function("Tas::set_fog_enabled")]
+fn set_fog_enabled(value: bool) {
+    let obj = unsafe { ObjectWrapper::new(UMyGameInstance::get_umygameinstance() as *mut UObject) };
+    let world_references = obj.get_field("WorldReferences").unwrap::<ObjectWrapper>();
+    let time_of_day = world_references.get_field("TimeOfDay").unwrap::<ObjectWrapper>();
+    let fog = time_of_day.get_field("Fog").unwrap::<ObjectWrapper>();
+    let component = fog.get_field("Component").unwrap::<ObjectWrapper>();
+    if value {
+        component.get_field("FogDensity").unwrap::<&Cell<f32>>().set(0.05);
+        component.get_field("DirectionalInscatteringExponent").unwrap::<&Cell<f32>>().set(0.);
+    } else {
+        component.get_field("FogDensity").unwrap::<&Cell<f32>>().set(0.);
+        component.get_field("DirectionalInscatteringExponent").unwrap::<&Cell<f32>>().set(0.);
+    }
+}
+#[rebo::function("Tas::set_sun_color")]
+fn set_sun_color(red: f32) {
```

Comment:
Maybe rename to `set_sun_redness` or `set_sun_color_red` to indicate the difference in arguments to `set_cloud_color` which takes RGB.